src/js/index.js

```diff
@@ -1,25 +1,37 @@
 import "../css/styles.scss";
 import "../css/reset.css";
 
-const $ = require("jquery");
 const Flipsnap = require("flipsnap");
 
-if (window.matchMedia("(max-width: 415px)").matches) {
-  $(function () {
-    Flipsnap("#about_slide", {
-      distance: 280,
+function setUpSlider() {
+  if (window.matchMedia("(max-width: 415px)").matches) {
+    const aboutSlide = Flipsnap("#about_slide", { distance: 275 }); // eslint-disable-line no-unused-vars
+    const aboutSelect = document.querySelectorAll(".about_select");
+
+    document.addEventListener("fspointmove", function () {
+      for (let i = 0; i < aboutSelect.length; i += 1) {
+        aboutSelect[i].classList.toggle("current");
+      }
     });
-  });
-  $(function () {
-    Flipsnap("#job_slide", {
-      distance: 280,
+
+    const jobSlide = Flipsnap("#job_slide", { distance: 275 }); // eslint-disable-line no-unused-vars
```

Comment:
適当にeslint無視するのやめろw
ちゃんと意味調べろwww

```suggestion
    Flipsnap("#job_slide", { distance: 275 });
```